modules/holodeckb2b-core/src/test/java/org/holodeckb2b/ebms3/handlers/inflow/FindPModesTest.java

```diff
@@ -135,15 +161,16 @@ public void testDoProcessing() throws Exception {
         agreement.setType(agreementRefType);
         pmode.setAgreement(agreement);
 
-        // todo this seems strange that we need to set the PMode id value separately
+        // todo It seems strange that we need to set the PMode id value separately
         // todo when it is contained within the agreement
-        // todo if we don't set it the value returned by userMessage.getPModeId() is null
+        // todo but if we don't set it the value returned by userMessage.getPModeId() is null now
         userMessage.setPModeId(pmodeId);
```

Comment:
Although this call does not affect the outcome of the test it might be better to remove as it is the handler's task to find the P-Mode and set it on the message unit.